MMCore/MMCore.cpp

```diff
@@ -2833,6 +2833,7 @@ void CMMCore::startSequenceAcquisition(long numImages, double intervalMs, bool s
 				throw CMMError(getCoreErrorText(MMERR_CircularBufferFailedToInitialize).c_str(), MMERR_CircularBufferFailedToInitialize);
 			}
 			cbuf_->Clear();
+         cbuf_->SetOverwriteData(false);
```

Comment:
This should be `cbuf_->SetOverwriteData(!stopOnOverflow)`.

The other two places are correct (we default to not overwriting, except for "continuous" sequence acquisition).

Reply:
Why would the other instance of `startSequenceAcquisition` not need this same change?

Reply:
Or perhaps did you mean that that the two `startSequenceAcquisition` functions should be `SetOverwriteData(!stopOnOverflow)`, while `startContinuousSequenceAcquisition` should be `SetOverwriteData(true)`?

Reply:
I meant that this particular line needs the change to `!stopOnOverflow`; the other two places where you call `SetOverwriteData` are already correct and don't need any change.

Reply:
I'm still confused by this. So you're saying that `startSequenceAcquisition(long numImages, double intervalMs, bool stopOnOverflow)` should be `!stopOnOverflow` but `startSequenceAcquisition(const char* label, long numImages, double intervalMs, bool stopOnOverflow)` can remain `false`? Why wouldn't these be the same? Isn't the difference between the methods just that you're providing the camera name in one case but using the core camera in the other? Am I missing something?

Reply:
Ah, sorry, my bad. You're right -- both should honor `!stopOnOverflow`. I probably missed that the second overload of `startSequenceAcquisition` had the `stopOnOverflow` parameter.

`startContinuousSequenceAcquisition` is tricky. It is _intended_ for Live view, which would be okay with overwriting, but the method name and documentation don't suggest this. However, `CCameraBase` implements the single-argument overload of `StartSequenceAcquisition` (called only by `startContinuousSequenceAcquisition`) by passing `false` for `stopOnOverflow`, so the actual behavior has always been to allow overwriting. The documentation could use a fix.

Reply:
Got it, thanks for clarifying!